scripts/meetingbaas.py

```
@@ -126,80 +143,89 @@ async def main(
+                    min_silence_duration_ms=100,
+                    min_volume=0.6,
                 ),
             ),
-            vad_audio_passthrough=True,
+            audio_in_passthrough=True,
             serializer=ProtobufFrameSerializer(),
-            timeout=300, 
+            timeout=300,
         ),
     )
+    log_and_flush(logging.INFO, f"[TRANSPORT] WebSocket transport initialized")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)